### ISSUE TITLE
Can't select a subtitle track

### ISSUE THREAD

**Issue by mkrsticch** [jellyfin/jellyfin-mpv-shim#261]
**Describe the bug**

1. Open up a file with more tnah 15 subtitle tracks in jellyfin-mpv-shim through jellyfin
2. Press "c" to open the config menu
3. Use the arrow keys or mouse to select "Change Subtitles"
4. You can see, choose and select only track you can see. You can't scroll down and select others

**Expected behavior**
We need a vertical scrooling here

**Screenshots**
[![mpv-shot0003.jpg](https://i.postimg.cc/NjZfw6vW/mpv-shot0003.jpg)](https://postimg.cc/0zdqYJQf)

**Desktop (please complete the following information):**
 - OS:  Windows 10
 - Version 2.0.2

**Comment by iwalton3** [comment]
Yes I am aware of this issue. That UI is a bit... annoying to fix. You can configure a list of languages to use in the config file to avoid having tons of languages you don't care about getting listed.

You configure the filter in `lang_filter`. The default is unknown, English, and Japanese. (`und,eng,jpn,mis,mul,zxx`) Then you can actually enable the filtering for subtitles and audio with the `lang_filter_sub` and `lang_filter_audio` settings.

At some point I likely will implement scrolling though. Or maybe even a UI that isn't a complete hack. (The way the UI works is it sends OSD messages to MPV and watches mouse coordinates.)

**Comment by mkrsticch** [comment]
> At some point I likely will implement scrolling though. Or maybe even a UI that isn't a complete hack. (The way the UI works is it sends OSD messages to MPV and watches mouse coordinates.)

Thank you for considering my request. Meanwhile, I will continue with `lang_filter` workaround.
